### ISSUE TITLE
Cairo-2 was not found

### ISSUE THREAD

**Issue by wimvandenborre** [zurie/LogicX-Push2#3]
Hey,

I tried to install and run but getting these errors:

Library/Python/3.9/lib/python/site-packages/cairocffi/__init__.py", line 45, in dlopen
    raise OSError(error_message)  # pragma: no cover
OSError: no library called "cairo-2" was found
no library called "cairo" was found
no library called "libcairo-2" was found

*ok solved by installing this: https://www.csestack.org/macbook-m1-oserror-no-library-called-cairo-pango-was-found/?fbclid=IwAR0knfHhf7plhHuHo5Ovgzqd_1fOztF3Mt6MSRQj0nuYB4iP0wUI3WCmNO4

Now in terminal i can see the buttons i press but push is not lid up or nothing else happens.. Any suggestions?

**Comment by wimvandenborre** [comment]
Did you try the latest files? Got updated 7 days ago with button_docs json file

**Comment by flippinloud** [comment]
I got the same, and it's because the Cairo libraries are missing. 
Try this "brew install cairo", that worked for me.

**Comment by moseleyluke** [comment]
hey, I have tried the above in terms of the cairo libraries but now am getting a whole bunch more errors. I just got the below, feel like I am sooooo close so any help is really really appreciated.

ERROR MESSAGE

Traceback (most recent call last):
  File "/Users/lukemoseley/Desktop/LogicX-Push2-master/app.py", line 6, in <module>
    import cairocffi as cairo
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/cairocffi/__init__.py", line 47, in <module>
    cairo = dlopen(
            ^^^^^^^
  File "/Library/Frameworks/Python.framework/Versions/3.11/lib/python3.11/site-packages/cairocffi/__init__.py", line 44, in dlopen
    raise OSError(error_message)  # pragma: no cover
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
OSError: no library called "cairo-2" was found
no library called "cairo" was found
no library called "libcairo-2" was found
cannot load library 'libcairo.so.2': dlopen(libcairo.so.2, 0x0002): tried: 'libcairo.so.2' (no such file), '/System/Volumes/Preboot/Cryptexes/OSlibcairo.so.2' (no such file), '/usr/lib/libcairo.so.2' (no such file, not in dyld cache), 'libcairo.so.2' (no such file), '/usr/lib/libcairo.so.2' (no such file, not in dyld cache).  Additionally, ctypes.util.find_library() did not manage to locate a library called 'libcairo.so.2'
cannot load library 'libcairo.2.dylib': dlopen(libcairo.2.dylib, 0x0002): tried: 'libcairo.2.dylib' (no such file), '/System/Volumes/Preboot/Cryptexes/OSlibcairo.2.dylib' (no such file), '/usr/lib/libcairo.2.dylib' (no such file, not in dyld cache), 'libcairo.2.dylib' (no such file), '/usr/lib/libcairo.2.dylib' (no such file, not in dyld cache).  Additionally, ctypes.util.find_library() did not manage to locate a library called 'libcairo.2.dylib'
cannot load library 'libcairo-2.dll': dlopen(libcairo-2.dll, 0x0002): tried: 'libcairo-2.dll' (no such file), '/System/Volumes/Preboot/Cryptexes/OSlibcairo-2.dll' (no such file), '/usr/lib/libcairo-2.dll' (no such file, not in dyld cache), 'libcairo-2.dll' (no such file), '/usr/lib/libcairo-2.dll' (no such file, not in dyld cache).  Additionally, ctypes.util.find_library() did not manage to locate a library called 'libcairo-2.dll'
lukemoseley@Lukes-Mac-mini LogicX-Push2-master %
